boards/cc2650stk/include/board.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2016 Leon George
+ * (who won't go after you should you copy any part of this file ;-)
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+/**
+ * @defgroup    boards_cc2650stk CC2650STK
```

Comment:
general idea: I think it would make sense to rename the board to `sensor-tag`. The more 'human readable' name the better and easier to use, and in this case this is the official name for the board, right?